encoder/libav_encoder.cpp

```diff
@@ -54,14 +58,18 @@ void encoderOptionsLibx264(VideoOptions const *options, AVCodecContext *codec)
 	av_opt_set(codec->priv_data, "mixed_ref", "0", 0);
 }
 
-const std::map<std::string, std::function<void(VideoOptions const *, AVCodecContext *)>> optionsMap =
-{
+const std::map<std::string, std::function<void(VideoOptions const *, AVCodecContext *)>> optionsMap = {
 	{ "h264_v4l2m2m", encoderOptionsH264M2M },
 	{ "libx264", encoderOptionsLibx264 },
 };
 
 } // namespace
 
+void LibAvEncoder::Signal()
+{
+	feed_encoder_frames = !feed_encoder_frames;
```

Comment:
I wonder if `feed_encoder_frames` should strictly be `feed_encoder_frames_`, just to maintain the convention about data member naming.